src/server/routes/pollRouter.test.ts

```diff
@@ -0,0 +1,122 @@
+import request = require("supertest");
+import app from "../app";
+import db, { Poll, PollInput } from "../models/database";
+
+describe("Test GET /api/polls", () => {
+  // adds test data before each test
+  beforeEach(() => {
+    db.insertPoll({
+      creatorName: "Jed",
+      description: "hey there",
+      options: ["bean bags"],
+      pollName: "test"
+    });
+    db.insertPoll({
+      creatorName: "James",
+      description: "testing again",
+      options: ["banana", "orange"],
+      pollName: "fruit"
+    });
+  });
+  afterEach(() => {
+    // cleans up test data after each test
+    db.removeAllPollsData();
+    db.resetCount();
+  });
+  test("GET should respond with 200 and json in body", async () => {
+    const response = await request(app)
+      .get("/api/polls")
+      .expect("Content-Type", /json/);
+    // const responseCleaned = response.body.map((poll: any) => {
+    //   return { name: poll.name, creator: poll.creator };
+    // });
+    expect(response.status).toBe(200);
+  });
```

Comment:
Doesn't need to be separate test here, one test to check `status=200` and response body is normal pattern